specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -383,6 +418,19 @@ model OptionalLoadTestConfig {
   duration?: int32;
 }
 
+@added(APIVersions.v2024_03_01_preview)
+@doc("Locust test configuration, in case kind is Locust")
+model LocustConfiguration {
+  @doc("Number of concurrent virtual users")
+  users?: int32;
+
+  @doc("Test run duration in seconds.")
+  durationInSeconds?: int32;
```

Comment:
not API. SDK may want you to use duration with `@encode(int32)` https://typespec.io/docs/standard-library/built-in-decorators#@encode